prog/postgres/postgres_resource_nexus.rb

```diff
@@ -86,7 +86,7 @@ def before_run
     # ago.
     postgres_resource.root_cert_1, postgres_resource.root_cert_key_1 = create_root_certificate(duration: 60 * 60 * 24 * 365 * 5)
     postgres_resource.root_cert_2, postgres_resource.root_cert_key_2 = create_root_certificate(duration: 60 * 60 * 24 * 365 * 10)
-    postgres_resource.server_cert, postgres_resource.server_cert_key = create_server_certificate
+    postgres_resource.server_cert, postgres_resource.server_cert_key = create_certificate
```

Comment:
If you're refactoring this, I suggest completely removing this state and moving the initial certificate creation into the assemble. This way, we don't have 2 states doing similar things and the dummy server cert wait is resolved.

Reply:
That makes sense. However it is a big refactor and touches to the places that I modified in subsequent PRs. I want to avoid creating lots of conflicts. After the HA PRs are merged, I'll make this change.